pkg/store/consul.go

```diff
@@ -81,8 +88,12 @@ func (s *Consul) setTimeout(time time.Duration) {
 }
 
 // SetEphemeralTTL sets the ttl for ephemeral nodes
-func (s *Consul) setEphemeralTTL(time time.Duration) {
-	s.ephemeralTTL = time
+func (s *Consul) setEphemeralTTL(ttl time.Duration) error {
```

Comment:
Don't know where to put the test for now to be honest as we only rely on mocking with `pkg/store`.